packages/app/src/app/pages/Dashboard/Components/UpgradeBanner/UpgradeBanner.tsx

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+  Banner,
+  Button,
+  Column,
+  Element,
+  Grid,
+  Icon,
+  IconNames,
+  Link,
+  Stack,
+  Text,
+} from '@codesandbox/components';
+import { useCreateCheckout, useDismissible } from 'app/hooks';
+import { dashboard } from '@codesandbox/common/lib/utils/url-generator';
+import track from '@codesandbox/common/lib/utils/analytics';
+
+const StyledTitle = styled(Text)`
+  font-size: 24px;
+  line-height: 32px;
+  letter-spacing: -0.019em;
+  margin: 0;
+`;
+
+// When flex wraps and the list of features is shown
+// above the call to action.
+const WRAP_WIDTH = 1332;
+
+type Feature = {
+  icon: IconNames;
+  label: string;
+};
+const FEATURES: Feature[] = [
+  {
+    icon: 'profile',
+    label: 'Up to 20 editors',
+  },
+  {
+    icon: 'npm',
+    label: 'Private NPM packages',
+  },
+  {
+    icon: 'sandbox',
+    label: 'Unlimited sandboxes',
+  },
+  {
+    icon: 'lock',
+    label: 'Advanced privacy settings',
+  },
+  {
+    icon: 'repository',
+    label: 'Unlimited repositories',
+  },
+  {
+    icon: 'server',
+    label: '6GB RAM, 12GB Disk, 4 vCPUs',
+  },
+];
+
+type UpgradeBannerProps = {
+  isAdmin: boolean;
+  teamId: string;
+};
+export const UpgradeBanner: React.FC<UpgradeBannerProps> = ({
+  isAdmin,
+  teamId,
+}) => {
+  const [checkout, createCheckout] = useCreateCheckout();
```

Comment:
I was wondering earlier about the checkout.error state, is this a normal case? should we handle it with an error message or something?

and lol, I was just looking to paste my PR link here when I saw you just commented on mine :D

Reply:
added an error message in comment: 6b503b633f5b460ab75c3ccc6f5aab8b07c15fd5.